website/src/assets/site.css

```
@@ -4958,16 +4962,21 @@ html:has(> body.docs-body)::-webkit-scrollbar-corner,
 }
 
 .stage2-workspace::before {
+  --stage2-aperture-radial-bg: radial-gradient(circle at var(--stage2-aperture-x, 50%) var(--stage2-aperture-y, 42%), rgba(59, 130, 246, var(--stage2-aperture-blue-alpha, 0.08)), transparent 34%);
+  --stage2-aperture-grid-y-bg: linear-gradient(rgba(9, 9, 11, var(--stage2-aperture-grid-alpha, 0.045)) 1px, transparent 1px);
+  --stage2-aperture-grid-x-bg: linear-gradient(90deg, rgba(9, 9, 11, var(--stage2-aperture-grid-alpha, 0.045)) 1px, transparent 1px);
+  --stage2-aperture-radial-pos: 50% 50%;
+  --stage2-aperture-grid-y-pos: 0px 17px;
+  --stage2-aperture-grid-x-pos: 17px 0px;
   content: "";
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)